VerilogCodingStyle.md

```diff
@@ -776,6 +776,15 @@ assign state_next = condition_b ?
     state;
 ```
 
+However, the nested ternary can be written in an unambiguous way in many
+cases. Parenthesis are then not required.
+
+&#x1f44d;
+```systemverilog {.good}
+assign a_or_b = condition_a ? a :
+                condition_b ? b : not_a_nor_b;
```

Comment:
Possibly indent the second line here?

Reply:
As just discussed: This is what we mostly use in our code base. I am thus leaving it.

Reply:
Personally I feel keeping the conditions lined up keeps things more readable